ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_searching.py

```diff
@@ -124,4 +124,44 @@ def test_numpy_argmin(
         axis=axis,
         keepdims=keep_dims,
         out=None
-    )
\ No newline at end of file
+    )
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=ivy_np.valid_numeric_dtypes),
+    keepdims=st.booleans(),
+    as_variable=helpers.array_bools(),
+    with_out=st.booleans(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.argmax"
+    ),
+    native_array=helpers.array_bools(),
+)
+def test_numpy_argmax(
+    dtype_and_x,
+    keepdims,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    input_dtype, x = dtype_and_x
+    ndim = len(ivy_np.shape(x, as_array=True))
```

Comment:
Could you make a helper function instead that samples ndim and axis inside it instead of doing it in `test_numpy_argmax` ?
Refer to below for reference: https://github.com/unifyai/ivy/blob/49c298f3e8bc02e6ef72e012faa53c969a7b08f9/ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_changing_number_of_dimensions.py#L10

Reply:
Also please resolve the conflicts, sorry the review process took so long. Will merge as soon as you do the changes.